build.gradle

```diff
@@ -81,10 +95,11 @@ ext {
   guiceVer = "3.0"
   jacksonVer = "2.6.3"
   junitVer = "4.11"
-  mesosVer = "1.0.0"
+  mesosVer = "1.1.0-SNAPSHOT"
```

Comment:
@gabrielhartmann That is correct, currently we can only deploy this on a bleeding edge cluster, which contains the correct `libmesos.so`. The plan right now is to keep this PR outstanding, and not merge into master. And, spin up long-running Kafka and Cassandra clusters and test out the HTTP API.
This can be merged into master, once mesos ships `1.1.0`.